test/integration/packer_pipeline_node/serverspec/packer_pipeline_node_spec.rb

```diff
@@ -51,10 +51,6 @@
   it { should be_executable }
 end
 
-describe command("scl enable python27 'openstack --version'") do
-  its(:exit_status) { should eq 0 }
-end
-
```

Comment:
This test can be adapted for the new setup, like `'/usr/local/bin/openstack --version'`

Reply:
That's already covered as an integration test in base::openstackclient

Reply:
What if someone accidentally removes it in this cookbook? or breaks it in some way?

Reply:
I suppose that's a good point, I'll adapt this test.